internal/controller/rabbitmq/transporturl_controller.go

```diff
@@ -304,7 +304,16 @@ func (r *TransportURLReconciler) reconcileNormal(ctx context.Context, instance *
 				Name:      userRef,
 				Namespace: instance.Namespace,
 			},
-			Spec: rabbitmqv1.RabbitMQUserSpec{RabbitmqClusterName: instance.Spec.RabbitmqClusterName, Username: instance.Spec.Username, VhostRef: vhostRef},
+			Spec: rabbitmqv1.RabbitMQUserSpec{
+				RabbitmqClusterName: instance.Spec.RabbitmqClusterName,
+				Username:            instance.Spec.Username,
+				VhostRef:            vhostRef,
+				Permissions: rabbitmqv1.RabbitMQUserPermissions{
+					Configure: ".*",
+					Read:      ".*",
+					Write:     ".*",
+				},
+			},
 		}
 		if err := controllerutil.SetControllerReference(instance, user, r.Scheme); err != nil {
```

Comment:
cannot remember why we do it here, I think we should just do it in the CreateOrUpdate, like e.g. inhttps://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/deployment/deployment.go#L70